### PR TITLE
refactor: rename `SortedSetPut` to `SortedSetPutElements`

### PR DESCRIPTION
**Pull request by malandis**
Adjusts request and response types appropriately.

### CODE REVIEW

**Review by cprice404 — APPROVED** [review]
this matches node.js so it looks right to me, but will defer to @poppoerika and @poppoerika to make sure that's how we're naming stuff for list requests etc.

File: examples/sortedset-example/main.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func main() {
 	// Put score for each element to set
 	// Using counter, element N has score N
 	for i := 1; i < 11; i++ {
-		_, err := client.SortedSetPut(ctx, &momento.SortedSetPutRequest{
+		_, err := client.SortedSetPut(ctx, &momento.SortedSetPutElementsRequest{
```
Comment (poppoerika) [inline examples/sortedset-example/main.go:31]: Do we want to call `client.SortedSetPutElements` here?

**Comment by pgautier404** [comment]
> this matches node.js so it looks right to me, but will defer to @poppoerika and @poppoerika to make sure that's how we're naming stuff for list requests etc.

This is in line with the API spec and other SDKs. As far as lists go, the equivalent operations would be `ListPushFront/Back` so no mention of `Element`s. We do have `ListRemoveValue` as opposed to `SetRemoveElement`, but that is consistent with the spec and other SDKs. This LGTM.

**Review by pgautier404 — APPROVED** [review]

File: examples/sortedset-example/main.go (hunk 1 above)
Comment (malandis) [reply examples/sortedset-example/main.go:31]: @poppoerika yes, but that will depend on the release version of the client, right?

Comment (malandis) [reply examples/sortedset-example/main.go:31]: Good eye. I didn't mean to update the examples.

**Review by pgautier404 — APPROVED** [review]

**Review by poppoerika — APPROVED** [review]

**Pull request merged**